tests/graphics/sceneviewer.cpp

```diff
@@ -501,47 +503,52 @@ TEST(ZincSceneviewer, description_io)
 {
 	ZincTestSetupCpp zinc;
 
+	std::cerr << "1.\n";
```

Comment:
There are a few of these what I presume to be debugging messages that need to be removed.

Reply:
Done.